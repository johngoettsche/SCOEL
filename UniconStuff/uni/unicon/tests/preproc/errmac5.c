/* test error message: invalid stringizing operand */

#define f(a) #b
f(1)
