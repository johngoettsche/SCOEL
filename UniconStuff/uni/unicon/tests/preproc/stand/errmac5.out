pp: File errmac5.c; Line 3: the # operator may only be applied to a macro argument
